buildout.cfg

```diff
@@ -29,7 +29,7 @@ clean-lines-exclude = ${buildout:directory}/src/collective/cover/static/js/vendo
 csslint = True
 csslint-bin = bin/csslint
 debug-statements = True
-flake8-ignore = B901,D001,E501,P001,T000
+flake8-ignore = B901,D001,E501,P001,T000,Q000
```

Comment:
Ignore Q000 it's no longer necessary: https://github.com/zheller/flake8-quotes/issues/55